packages/ap-storage/alliance_platform/storage/models.py

```diff
@@ -0,0 +1,2 @@
+# this is just here so the model can be detected for migrations
+from alliance_platform.storage.async_uploads.models import AsyncTempFile as AsyncTempFile
```

Comment:
what's with the `as AsyncTemplFile`?

Reply:
Ruff check recommended for isolated imports in the `__init__` files, so I figured I'd do the same here.

Reply:
It suppresses "imported but unused" errors.